swift.html.markdown

```diff
@@ -74,11 +76,17 @@ if someOptionalString != nil {
     if someOptionalString!.hasPrefix("opt") {
         print("has the prefix")
     }
-    
+
     let empty = someOptionalString?.isEmpty
 }
 someOptionalString = nil
 
+/*
+    To get the underlying type from an optional, you unwrap it using the 
+    force unwrap operator (!). Only use the unwrap operator if you're sure 
+    the underlying value isn't nil.
```

Comment:
True, good point. I was more trying to explain what ! was used for in swift, rather than implying it should be the only way to unwrap an optional. Maybe something along the lines of:

"Trying to use ! to access a non-existent optional value triggers a runtime error. Always make sure that an optional contains a non-nil value before using ! to force-unwrap its value."  

instead. Thoughts?